CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.7.7] - 2024-06-20
 
 - Added warning about using private endpoints with old versions.
```

Comment:
Should we add a comment about the other PR that goes into this release: "services map to private_endpoint_services "

Reply:
It's there if you expand on the file.

Reply:
It's a functional change so I added it to the `Added` section.